packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalCardExecuted.tsx

```diff
@@ -32,20 +33,20 @@ export function WithdrawalCardExecuted({ tx }: { tx: MergedTransaction }) {
   }, [])
 
   const balance = useMemo(() => {
-    if (!arbTokenBridge || !arbTokenBridge.balances) {
+    if (!ethL1Balance || !erc20L1Balances) {
       return null
     }
 
     if (tx.asset === 'eth') {
-      return ethBalance
+      return ethL1Balance
     }
 
     if (!tx.tokenAddress) {
       return null
     }
 
-    return arbTokenBridge.balances.erc20[tx.tokenAddress]?.balance
-  }, [ethBalance, tx, arbTokenBridge])
+    return erc20L1Balances?.[tx.tokenAddress.toLowerCase()]
```

Comment:
Just wondering, can we use `useTokenBalances` hook here to achieve this and not worry about calculating balance everywhere?